hugo/layouts/partials/home/hero.html

```diff
@@ -0,0 +1,27 @@
+<header class="home-hero">
+    <div class="home-hero__wrap">
+        <div class="home-hero__info">
+            <h1>{{ .title }}</h1>
+            <p>{{ .desc }}</p>
+            <div class="home-hero__info__links">
```

Comment:
I do not see the differences:

hero:
![image](https://user-images.githubusercontent.com/2437584/26924872-d7488902-4c47-11e7-9b45-5f86a816863e.png)

roadmap:
![image](https://user-images.githubusercontent.com/2437584/26924823-a39c395a-4c47-11e7-9356-51850c5e8308.png)

And if there are differences, it can be handled with the same class:

```
.home-hero .btn-pill {
    /* custom styles for hero pills */
}
.roadmap .btn-pill {
    /* custom styles for roadmap pills */
}
```

And looking at the docs, it is said that:
>_"Buttons are regular buttons"_

roadmap:
![image](https://user-images.githubusercontent.com/2437584/26925156-ed4b05e4-4c48-11e7-8fdd-a1e0363854fc.png)

hero:
![image](https://user-images.githubusercontent.com/2437584/26925099-a98bcbcc-4c48-11e7-976a-f18e2ae07aa9.png)